tests/test_nn/test_init.py

```diff
@@ -86,3 +88,21 @@ class XavierUniformNormTestCase(NormalizationMixin, cases.InitializerTestCase):
     """Tests for Xavier Glorot uniform initialization + normalization."""
 
     initializer = staticmethod(pykeen.nn.init.xavier_uniform_norm_)
+
+
+class LabelBasedInitializerTestCase(cases.InitializerTestCase):
+    """Tests for label-based initialization."""
+
+    def setUp(self) -> None:
+        """Prepare for test."""
+        from pykeen.datasets import get_dataset
+
+        dataset = get_dataset(dataset="nations")
+        try:
+            self.initializer = pykeen.nn.init.LabelBasedInitializer.from_triples_factory(
+                triples_factory=dataset.training,
+                for_entities=True,
+            )
+        except ImportError as error:
+            raise SkipTest from error
```

Comment:
this will essentially skip the test in CI. If we wanted to have this running, we would need to mock some parts of the transformer library (and the fact that it is missing :smile: )